### ISSUE TITLE
timeout error and permission error (window + wsl2)

### ISSUE THREAD

**Issue by NAMU1105** [cypress-io/cypress#19947]
### Current behavior

can not open cypress due to timeout error + permission error.
(I even changed the default waiting time from 30000 to 10000) 

People say that I have to clear cache etc. So I followed their guides and encountered the permission error while doing that.

Therefore I tried to solve the permission issue with chmod 777 -R and chown,
But it still does not work.

![image](https://user-images.githubusercontent.com/47317129/151521271-13d0db4d-244f-4c0f-8d43-2d2a05e41a4d.png)

### Desired behavior

Cypress should work as it should.

### Test code to reproduce

yarn run cypress open

### Cypress Version

9.3.1

### Other

It was originally worked fine.

But I haven't used Cypress for a month.
And when I hit run command today I've faced this issue.

Please give me some help.

**Comment by tbiethman** [comment]
@NAMU1105 Can you provide the full error message and other relevant system details? 

We've had similar issues come in recently: https://github.com/cypress-io/cypress/issues/19457 https://github.com/cypress-io/github-action/issues/446

**Comment by NAMU1105** [comment]
> @NAMU1105 Can you provide the full error message and other relevant system details (node/yarn/OS versions)? Did you recently update your version of Cypress?
> 
> We've had similar issues come in recently: #19457 [cypress-io/github-action#446](https://github.com/cypress-io/github-action/issues/446)


Will do it on Monday! (I'm on my lunar new year holiday vacation)

**Comment by NAMU1105** [comment]
Hey! I solved this problem today.

I originally have used cypress@9.2.0

But after I update cypress to version 9.3.0, I faced the problem.




So, I guess maybe the error is derived from the newer version issue 

or...
I hit the 'npm uninstall cypress --save-dev' just in case 
(I use yarn, so I was just trying to make sure every cypress-related stuff is erased from my computer.)

Maybe I somehow installed the cypress with npm (even though I use yarn) and that might cause me trouble.

Anyway, thanks for the concern and I'll close this issue.
